Infrastructure-Scripts/Add-AppServiceIpException.ps1

```diff
@@ -25,42 +25,43 @@ Param (
     [IPAddress]$IpAddress,
     [Parameter(Mandatory = $true)]
     [ValidateNotNull()]
-    [String]$ResourceName,
-    [Parameter(Mandatory = $false)]
-    [Switch]$UpdateReleaseName
+    [String[]]$ResourceName,
```

Comment:
ResourceNames

Reply:
Updated to reflect ability to pass in multiple resource names